generators/server/templates/src/main/resources/config/cql/changelog/_create-tables.cql

```diff
@@ -1,12 +1,3 @@
-CREATE TABLE IF NOT EXISTS schema_version (
```

Comment:
same comment: maybe keep here, in this file (but this file has been moved to changelog/)